static/app/components/profiling/profilingOnboardingSidebar.tsx

```diff
@@ -312,7 +312,11 @@ function ProfilingOnboardingContent(props: ProfilingOnboardingContentProps) {
   };
 
   const doc = docs.profilingOnboarding ?? docs.onboarding;
-  const steps = [...doc.install(docParams), ...doc.configure(docParams)];
+  const steps = [
+    ...doc.install(docParams),
+    ...doc.configure(docParams),
+    ...doc.verify(docParams),
```

Comment:
Very good point, I was under the impression that the existence of the `profilingOnboarding` attribute in the Docs object indicates that it will always be used and separately created, but we [fallback to the normal docs](https://github.com/getsentry/sentry/blob/43dc6644c47b4b7afe02fae128e4ee63a60b1dec/static/app/components/profiling/profilingOnboardingSidebar.tsx#L314) in case there is no separate profiling onboarding.